lib/ansible/module_utils/docker_common.py

```diff
@@ -83,16 +83,16 @@
 DEFAULT_TIMEOUT_SECONDS = 60
 
 DOCKER_COMMON_ARGS = dict(
-    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST),
-    tls_hostname=dict(type='str', default=DEFAULT_TLS_HOSTNAME),
-    api_version=dict(type='str', aliases=['docker_api_version'], default='auto'),
+    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST, fallback=(env_fallback, 'DOCKER_HOST')),
+    tls_hostname=dict(type='str', default=DEFAULT_TLS_HOSTNAME, fallback=(env_fallback, 'DOCKER_TLS_HOSTNAME')),
+    api_version=dict(type='str', aliases=['docker_api_version'], default='auto', fallback=(env_fallback, 'DOCKER_API_VERSION')),
     timeout=dict(type='int', default=DEFAULT_TIMEOUT_SECONDS),
```

Comment:
Ah, sorry, I forgot one more: `timeout` is missing the `DOCKER_TIMEOUT` fallback.